### DISCUSSION TITLE
could I use the intense packet generator to flood a wifi drone?

### DISCUSSION THREAD

**Discussion by Ir0n2** [dannagle/PacketSender#293] [Q&A]
Gathering dust in my closet is a drone, one of those ones with a wifi camera. Would I be able to flood the network that the camera uses with the intense packet generator? I'll probably try it anyway. Does anybody have experience with similar projects?

**Comment by dannagle** [comment]
The intense packet generator in the GUI is pretty primitive. If you need more options, take a look at the command-line version:

https://github.com/dannagle/PacketSender#cliintensetraffic
